assets/package.json

```diff
@@ -0,0 +1,140 @@
+{
+	"scripts": {
+		"build": "npm run build:production",
+		"prebuild:dev": "npm run remove-dist",
+		"build:dev": "webpack --mode=development --devtool cheap-module-eval-source-map --output-pathinfo",
+		"prebuild:production": "npm run remove-dist",
+		"build:production": "webpack --mode=production",
+		"build:test": "npm run build:production -- --include-tests",
+		"watch": "npm run build:dev -- --watch",
+		"remove-dist": "rimraf ../dist"
+	},
+	"browserslist": [
+		"extends @wordpress/browserslist-config"
+	],
+	"dependencies": {
+		"@lit-labs/react": "^1.2.0",
+		"@material-ui/core": "^4.11.4",
+		"@material/button": "^2.3.0",
+		"@material/checkbox": "^2.3.0",
+		"@material/dialog": "^2.3.0",
+		"@material/form-field": "^2.3.0",
+		"@material/layout-grid": "^0.41.0",
+		"@material/linear-progress": "^1.1.0",
+		"@material/list": "^2.3.0",
+		"@material/menu": "^2.3.0",
+		"@material/radio": "^2.3.0",
+		"@material/react-card": "^0.15.0",
+		"@material/react-chips": "^0.15.0",
+		"@material/react-dialog": "^0.15.0",
+		"@material/react-select": "^0.15.0",
+		"@material/react-tab-bar": "^0.15.0",
+		"@material/react-text-field": "^0.15.0",
+		"@material/ripple": "^2.3.0",
+		"@material/select": "^2.3.1",
+		"@material/switch": "^2.3.0",
+		"@material/textfield": "^2.3.1",
+		"@material/theme": "^1.1.0",
+		"@material/web": "^1.0.0-pre.9",
+		"@popperjs/core": "^2.9.2",
+		"@reach/combobox": "^0.10.5",
+		"@react-hook/merged-ref": "^1.3.0",
+		"@react-hook/window-size": "^3.0.7",
+		"@wordpress/api-fetch": "^3.18.0",
+		"@wordpress/compose": "^3.25.3",
+		"@wordpress/data": "^4.27.3",
+		"@wordpress/dom-ready": "^2.10.0",
+		"@wordpress/element": "^2.20.3",
+		"@wordpress/i18n": "^3.14.0",
+		"@wordpress/icons": "^2.9.0",
+		"@wordpress/keycodes": "^3.2.4",
+		"@wordpress/url": "^3.1.1",
+		"accessible-autocomplete": "^2.0.3",
+		"classnames": "^2.2.6",
+		"clipboard-copy": "^3.1.0",
+		"compare-versions": "^3.6.0",
+		"dompurify": "^2.5.8",
+		"element-closest": "^3.0.2",
+		"focus-trap-react": "^10.0.0",
+		"formdata-polyfill": "^3.0.20",
```

Comment:
One more polyfill that we don't use, right? Let's get rid of it.

Reply:
LGTM, removed.